features/dnsresolver/netty/src/main/java/org/opennms/netmgt/dnsresolver/netty/NettyDnsResolver.java

```diff
@@ -162,6 +169,12 @@ public void init() {
                 .onCallNotPermitted(e -> {
                     lookupsRejectedByCircuitBreaker.mark();
                 });
+
+        if (breakerEnabled) {
+            circuitBreaker.transitionToClosedState();
+        } else {
+            circuitBreaker.transitionToDisabledState();
```

Comment:
The lookup statistics i.e. `lookupsSuccessful` and `lookupsFailed` are no longer tracked when the circuit breaker are disabled (since we use the circuit breaker callbacks to track these).

We don't necessarily need to fix this, but it may be worth mentioning in the docs.

Reply:
Documentation note added.